batcher/aligned-batcher/src/lib.rs

```diff
@@ -984,13 +1007,25 @@ impl Batcher {
             ));
         };
 
+        let Some(current_total_fees_in_queue) = batch_state_lock
+            .get_user_total_fees_in_queue(&proof_submitter_addr)
+            .await
+        else {
+            error!("User state of address {proof_submitter_addr} was not found when trying to update user state. This user state should have been present");
+            std::mem::drop(batch_state_lock);
+            return Err(BatcherError::AddressNotFoundInUserStates(
+                proof_submitter_addr,
```

Comment:
Ditto

Reply:
not here , it returns error, and the caller sends the msg